elements/rh-spinner/rh-spinner.css

```diff
@@ -0,0 +1,81 @@
+:host {
+  display: inline-block;
+  aspect-ratio: 1 / 1;
+  text-align: center;
+  --rh-spinner--AnimationDuration: 1.2s;
```

Comment:
these should
1. conform to the [naming convention](https://github.com/RedHat-UX/red-hat-design-system/wiki/CSS-Styles#tokens-and-naming-conventions), or even better be replaced with token values e.g. `var(--rh-length-sm, 6px)`
2. be [provided as defaults instead of set on the host](https://github.com/RedHat-UX/red-hat-design-system/wiki/CSS-Styles#using-css-variables)

or alternately, if they are intended to be 'private' i.e. not meant for user customization, they can stay assigned to the :host but should use the [private var naming convention](https://github.com/RedHat-UX/red-hat-design-system/wiki/CSS-Styles#private-variables)